.github/workflows/e2etests.yml

```diff
@@ -237,3 +237,119 @@ jobs:
           commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
           commit_options: '--signoff'
           commit_message: '[Docs] Test status of adapter'
+
+  SetPatternfileForServicemeshAndSampleApp:
+    runs-on: ubuntu-latest
+    outputs:
+      sm_version: ${{ steps.gettag.outputs.release }}
+      adapter_version:  ${{ env.version }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Get version of adapter
+        run: |
+            if [ ${{ github.event_name }} == "release" ];then
+                echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+            else 
+                echo "version=edge" >> $GITHUB_ENV
+            fi
+      - name: Get latest release tag
+        id: gettag
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: istio/istio
+          excludes: prerelease, draft, edge 
+      - name: Change service mesh version in patternfile
+        run: | 
+           yq e -i '.services.istioinstallation.settings.istioVersion="${{ steps.gettag.outputs.release }}"' ./.github/install/deploywithsampleapp.yaml 
+           cat ./.github/install/deploywithsampleapp.yaml
+      - name: Uploading file
+        uses: actions/upload-artifact@v2
+        with:
+          name: patternfile
+          path: ./.github/install/deploywithsampleapp.yaml 
+
+  TestIstioForServicemeshAndSampleApp:
+    needs: SetPatternfileForServicemeshAndSampleApp
+    uses: meshery/meshery/.github/workflows/testadapters.yaml@master
+    with:
+      expected_pods: istiod,details73de2529,productpageb6cd6107,ratings8229a44f,reviews-v13837c681,reviews-v27200aff4,reviews-v35f86bb6a
```

Comment:
Will expected_pods have the same suffix on every test run? productpage**_b6cd6107_**, ratings**_8229a44f_**, reviews-v1**_3837c681_**

Reply:
Please remove the suffix from here as they will be different across the runs

Reply:
I checked, this part of the suffix is constant between all runs. Still remove?

Reply:
yes, just have details, productpage,etc